### PR TITLE
etcdserver, backend: configure mmap size based on quota

### PR DESCRIPTION
**Pull request by heyitsanthony**
Mechanism for permitting mmaps > 10GB.

### CODE REVIEW

File: etcdserver/quota.go [hunk 1]
```diff
@@ -57,11 +65,10 @@ func NewBackendQuota(s *EtcdServer) Quota {
...
 	if s.Cfg.QuotaBackendBytes == 0 {
 		// use default size if no quota size given
-		return &backendQuota{s, backend.DefaultQuotaBytes}
+		return &backendQuota{s, DefaultQuotaBytes}
 	}
-	if s.Cfg.QuotaBackendBytes > backend.MaxQuotaBytes {
-		plog.Warningf("backend quota %v exceeds maximum quota %v; using maximum", s.Cfg.QuotaBackendBytes, backend.MaxQuotaBytes)
-		return &backendQuota{s, backend.MaxQuotaBytes}
+	if s.Cfg.QuotaBackendBytes > MaxQuotaBytes {
+		plog.Warningf("backend quota %v exceeds maximum recommended quota %v", s.Cfg.QuotaBackendBytes, MaxQuotaBytes)
```
Comment (xiang90) [inline etcdserver/quota.go]: so we change the behavior here to allow user to use more than 8gb?

**Comment by xiang90** [comment]
LGTM

**Comment by fanminshi** [comment]
lgtm

File: etcdserver/quota.go (hunk 1 above)
Comment (heyitsanthony) [inline etcdserver/quota.go]: yes

**Pull request merged**